docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>From this sentence, it is unclear why Apartment should inherit from House. For example, what are the pros and cons with respect to Apartment having a House as a property?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>